pkgs/development/libraries/libfive/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, eigen3_3, zlib
+, libpng, boost, qt5, guile
+}:
+
+stdenv.mkDerivation rec {
+  name = "libfive-${version}";
+  version = "755f7fed";
+
+  src = fetchFromGitHub {
+    owner  = "libfive";
+    repo   = "libfive";
+    rev    = "${version}";
+    sha256 = "1cjvmfsjbc3k5zfp22zccmjx0b2rmm2a0vm555f3bqj81yw9v7ps";
+  };  
+  buildInputs = [ cmake pkgconfig eigen3_3 zlib libpng
+                  boost qt5.qtimageformats guile ];
+
+  enableParallelBuilding = true;
+
+  # Rename "Studio" binary to "libfive-studio" to be more obvious
+  postFixup = ''
+    mv "$out/bin/Studio" "$out/bin/libfive-studio"
```

Comment:
Wouldn't this confuse people who used it in other package managers? I think either not changing the name at all, or at least symlinking so both names are usable, would be better.

Reply:
If it's no problem having a generically-named "Studio" binary, sure, I'll do that.